market.go

```diff
@@ -238,6 +241,15 @@ func (c *Client) formatHistoricalData(inp historicalDataReceived) ([]HistoricalD
 		}
 
 		volume = int(v)
+		// Did we get OI?
+		if len(i) > 6 {
+			// Assert OI
+			OIT, ok := i[6].(float64)
```

Comment:
OI is an int right, why not just do `i[6].(int)`?

Reply:
Hmm, I guess the contributor must have copied the way we parse volume.

Seems like it errors out.

```
		volume, ok = i[5].(int)
		if !ok {
			return data, NewError(GeneralError, fmt.Sprintf("Error decoding response `volume`: %v (%T)", i[5], i[5]), nil)
		}
```

returns the following error:

```
Error:      	Expected nil, but got: kiteconnect.Error{Code:500, ErrorType:"GeneralException", Message:"Error decoding response `volume`: 163275 (float64)", Data:interface {}(nil)}
```